build_barback/lib/src/transformer/transformer.dart

```diff
@@ -121,6 +122,10 @@ class _TransformAssetReader implements AssetReader {
   @override
   Future<String> readAsString(build.AssetId id, {Encoding encoding: UTF8}) =>
       transform.readInputAsString(toBarbackAssetId(id), encoding: encoding);
+
+  @override
+  Iterable<build.AssetId> findAssets(Glob glob) => throw new UnsupportedError(
+      'findAssets cannot be used within a Transformer');
```

Comment:
Lets add a link to the github repo where they can file a bug about this if they need it.